src/algolia-components/search-input/index.js

```diff
@@ -19,3 +23,18 @@ const SearchInputRainbowAlgoliaAdapter = ({ className, currentRefinement, refine
 const SearchInputRainbowAlgolia = connectSearchBox(SearchInputRainbowAlgoliaAdapter);
 
 export default SearchInputRainbowAlgolia;
+
+SearchInputRainbowAlgoliaAdapter.propTypes = {
+    currentRefinement: PropTypes.any,
+    refine: PropTypes.any,
```

Comment:
let's only put here props that we owns, since refine and currentRefinement are from algolia then remove them in proptypes